src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx

```diff
@@ -113,6 +113,7 @@ export default function MeasureInformation() {
         .updateMeasure(newMeasure)
         .then(() => {
           setMeasure(newMeasure);
+          updateMeasure(newMeasure);
```

Comment:
Having `.updateMeasure().then(() => updateMeasure())` is a little confusing.  

Though, that might be a function of how changes are displayed in the PR. Reading the whole promise chain from the top makes it easier to understand.

Reply:
113: update measure is a method of the measureStore that updates the current measure
116: is a function that sends the actual update request. I'm unsure if there's a better way to name it